### ISSUE TITLE
Continuous capture 

### ISSUE THREAD

**Issue by JellyBeans** [google/grafika#24]
In continuous capture Activity, when press power button and press it again, the continuous capture does not work. The reason seems that the Surfaceview has not been destoried. So we need do the init in callback surfaceCreated Again after onResume if the sruface is still alive.

**Comment by fadden** [comment]
I'm not currently set up to debug this, but I think your assessment is correct.  It's tearing everything down in `onPause()`, so it'll need to set it back up in `onResume()` if the Surface wasn't destroyed.

**Comment by JellyBeans** [comment]
Yeah!  The explanation is in you post  http://source.android.com/devices/graphics/architecture.html Appendix B: SurfaceView and the Activity Lifecycle ): .
